.github/workflows/run-tests.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - if: steps.cache-ophys-datasets.outputs.cache-hit == false
         name: Install and configure AWS CLI
         run: |
-          pip install awscli==1.25.27
+          pip install awscli==1.29.56
```

Comment:
My suggestion was to completely remove the step from this workflow so you don't run into this problem again and to keep the steps more modular in general

Reply:
Ok, I must be misuderstanding something bc when I remove the awscli call from the run-tests workflow, I get an error: `Failed: No folder found in location: ophys_testing_data!`

See [Action #230](https://github.com/catalystneuro/roiextractors/actions/runs/6342979572)

Reply:
What if we just removed `update-testing-data.yml` since it's redundant?

Reply:
Or is the whole point of that workflow for debugging?

Reply:
> Ok, I must be misuderstanding something bc when I remove the awscli call from the run-tests workflow, I get an error:

Because you didn't just remove the AWS CLI part, but also the caching (which is the thing that has the folder): https://github.com/catalystneuro/roiextractors/blob/ce624f8dfc07bd0e89dbeb2a59e4ffc0125e082e/.github/workflows/run-tests.yml#L48-L65

> What if we just removed update-testing-data.yml since it's redundant?

Other way around

> Or is the whole point of that workflow for debugging?

Not for debugging, but as the primary way of updating the caches for all platform/python versions from a common source while also keeping their environments (downloading data vs. running tests) completed isolated to avoid stuff like this forever into the future

Reply:
It's only these lines: https://github.com/catalystneuro/roiextractors/blob/ce624f8dfc07bd0e89dbeb2a59e4ffc0125e082e/.github/workflows/run-tests.yml#L57-L65

That you would have to remove; and perhaps shift that CRON trigger on the update job when relying on the scheduling

Reply:
Ahh, I see thanks -- I didn't catch that the cache action grabs the files if they've already been downloaded.